.github/workflows/test.yml

```diff
@@ -258,3 +257,4 @@ jobs:
         php build/moodle-plugin-ci.phar phpunit --verbose --coverage-text --fail-on-warning
         php build/moodle-plugin-ci.phar behat --profile default
         php build/moodle-plugin-ci.phar behat --profile chrome
+        php build/moodle-plugin-ci.phar behat --profile firefox --tags="@local_ci&&~@app"
```

Comment:
Same than above... I imagine I'll understand later when arriving to the code, but right now it doesn't fit in my brain, hence annotating.